server/database_initialization/geocoder.js

```diff
@@ -0,0 +1,45 @@
+import NodeGeocoder from 'node-geocoder';
+import geocoderOptions from '/server/database_initialization/geocoderSettings'
```

Comment:
re: Ian's comment on slack. is there anyway this can be put into `settings.json`? then import Meteor, and use `Meteor.settings.[json_path_to_setting]` to reference the relevant settings. that way less settings files to keep track of offline.